packages/components/molecules/f-rating/src/tenants/en-GB.js

```diff
@@ -1,4 +1,8 @@
 export default {
     locale: 'en-GB',
-    text: 'I am a VRating Component (GB)'
+    text: 'I am a VRating Component (GB)',
+    ratings: {
+        starsDescription: 'stars out of',
```

Comment:
we can use vue i18n pluralization here:
https://kazupon.github.io/vue-i18n/guide/pluralization.html